jsonpatch.py

```diff
@@ -56,7 +56,7 @@
 
 # Will be parsed by setup.py to determine package metadata
 __author__ = 'Stefan Kögl <stefan@skoegl.net>'
-__version__ = '1.24'
+__version__ = '1.27'
```

Comment:
Please do not change the version number as part of the pull-reques. That will be done when a new release is prepared.

Reply:
\+